app/components/Views/RevealPrivateCredential/RevealPrivateCredential.tsx

```diff
@@ -296,67 +299,69 @@ const RevealPrivateCredential = ({
     }
   };
 
-  const renderTabView = (privCredentialName: string) => {
+  useEffect(() => {
     Device.isAndroid() &&
       Device.getDeviceAPILevel().then((apiLevel) => {
         if (apiLevel < AppConstants.LEAST_SUPPORTED_ANDROID_API_LEVEL) {
```

Comment:
`clipboardEnabled` already defaults to `false`. so why not reverting the test and use ternary?
```suggestion
        setClipboardEnabled (apiLevel >= AppConstants.LEAST_SUPPORTED_ANDROID_API_LEVEL)
```